src/lib/cli/browserslist-generator.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @fileoverview Generates a valid browserslist config.
+ */
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+import * as inquirer from 'inquirer';
+import * as browserslist from 'browserslist';
+import * as logger from '../utils/logging';
+import { debug as d } from '../utils/debug';
+
+const debug: debug.IDebugger = d(__filename);
+
+export const generateBrowserslistConfig = (): Promise<string[]> => {
```

Comment:
`Promise<Array<string>>`

It is just to be consistent with the style. we are using that style everywhere in the code. There are more places where you have this (or similar).

Reply:
@sarvaje Fixed.